### ISSUE TITLE
Training loss and acc/auc curve

### ISSUE THREAD

**Issue by wwx13** [microsoft/AMOS#1]
Hi,
I'm using amos now. My amos  model (small size, discriminator ) have a low recall (70-80% percision while 40% recall).  60% mlm acc of generator. 
I would just like to ask if you can post the loss of both base and large models (or even share the loss training curve, acc curve or auc curve ) so that i have a kind of reference point when training own models. This will help me a lot! 

Thank u.

**Comment by yumeng5** [comment]
Hi @wwx13 

We only have the logs for base models. The logs contain the following metrics which I believe are relevant to your question.
* Discriminator accuracy on all tokens:
<img width="424" alt="image" src="https://user-images.githubusercontent.com/22209037/169680363-0bfd3476-05e5-4daf-97c1-6eb12800952d.png">

* Discriminator accuracy on non-replaced tokens:
<img width="426" alt="image" src="https://user-images.githubusercontent.com/22209037/169680359-d058805e-d0fb-4e16-ac33-f5e5972a0858.png">

* Discriminator accuracy on replaced tokens:
<img width="432" alt="image" src="https://user-images.githubusercontent.com/22209037/169680296-25bb04ad-b143-42af-9acd-a34a1a23426b.png">

* Generator token replace rate (defined to be the portion of replaced tokens by the generator out of all tokens):
<img width="431" alt="image" src="https://user-images.githubusercontent.com/22209037/169680308-468a0d57-3bbc-498c-9e7a-c37a1bebfe1e.png">

I hope these are helpful. Let me know if you have further questions.

Thanks,
Yu
